common/onepanel/base/add-label.yaml

```diff
@@ -3,4 +3,5 @@ kind: Namespace
 metadata:
   name: $(defaultNamespace) # replace with relevant namespace
   labels:
-    istio-injection: enabled
\ No newline at end of file
+    istio-injection: enabled
+    onepanel.io/enabled: "true"
```

Comment:
@rushtehrani This is to match the changes in Core right? But in Core this is an environment variable, should Core be hard coded to match this?

Reply:
@Vafilor are you talking about `labelPrefix` in Core?

Reply:
@rushtehrani Yes, `labelKeyPrefix`

Reply:
Yes, we should update Core to in the least grab it from an environment variable but set a default of "onepanel.io/". This will most likely never change.

Reply:
Cool, sounds good.

Reply:
@Vafilor see https://github.com/onepanelio/core/pull/39